content/en/integrations/guide/source-code-integration.md

```diff
@@ -33,8 +27,8 @@ Datadog Agent 7.35.0 or higher is required.
 To map telemetry data with your source code:
 
 1. Add `git.commit.sha` and `git.repository_url` tags to your containers, or directly on your telemetry.
-2. Upload metadata about your git repository by running [`datadog-ci git-metadata upload`][1] in your CI pipeline.
-3. Optionally, [install a GitHub App][2] to display inline source code snippets.
+2. a. Install Datadog's [GitHub Apps integration][2] to display inline source code snippets.
+2. b. Alternatively, upload metadata about your git repository by running [`datadog-ci git-metadata upload`][1] in your CI pipeline.
```

Comment:
I think it's a bit confusing to have
```
a. Do xxx to display inline code snippets
b. Alternatively do yyy
```
This might let people think that doing `yyy` will allow showing code snippets.

What do you think about having two "tabs" (like we have for the different container setups), one for "GitHub" and another for "Other Git Providers" ?

Reply:
I've updated it with the tabs you suggested